scripts/scil_volume_stats_in_labels.py

```diff
@@ -11,15 +11,18 @@
 """
```

Comment:
In the description above, can you change input map for input metric (s)?. Then, maybe something like "if more than one metric maps are provided, statistics are computed separately for each".